### ISSUE TITLE
[bug] 3.1 breaks serial ids in schemas

### ISSUE THREAD

**Issue by tomasc** [crashtech/torque-postgresql#77]
It modifies

```ruby
create_table "my_table", id: :serial, force: :cascade do |t|
```

to

```ruby
create_table "my_table", id: :integer, default: -> { "nextval('my_table_id_seq'::regclass)" }, force: :cascade do |t|
```

where the default value does not work (probably) since the table does not exist yet.

**Comment by crashtech** [comment]
Can you try out the version that is now available on master?

**Comment by tomasc** [comment]
same result

**Comment by tomasc** [comment]
@crashtech yep, the current version on master does the trick!

**Comment by crashtech** [comment]
Awesome! I'll be launching a patch version later today then.

**Comment by tomasc** [comment]
thank you for such a quick fix, much appreciated!

**Comment by crashtech** [comment]
Done! It's now available on version 3.2.1

**Comment by tomasc** [comment]
thank you! updated the gem all my tests pass again
